src/main/scala/com/typesafe/sbt/packager/docker/DockerPlugin.scala

```diff
@@ -273,11 +274,21 @@ object DockerPlugin extends AutoPlugin {
     def publishTask =
       Def.task {
         val _ = publishLocal.value
-        val alias = dockerAliases.value
         val log = streams.value.log
-        val execCommand = dockerExecCommand.value
+        val alias = dockerAliases.value
+        val context = stage.value
+        val multiplatform = dockerBuildxPlatforms.value.nonEmpty
+        val execCommand =
+          if (multiplatform)
+            dockerExecCommand.value ++ Seq(
+              "buildx",
+              "build",
+              s"--platform=${dockerBuildxPlatforms.value.mkString(",")}",
+              "--push"
+            ) ++ dockerBuildOptions.value :+ "."
+          else dockerBuildCommand.value
```

Comment:
@muuki88 @dswiecki This line introduced a bug. Here's a fix: https://github.com/sbt/sbt-native-packager/pull/1533

Reply:
yes it should be dockerExecCommand, thank for quick response 👍